### DISCUSSION TITLE
Compatibility with Compute Module 3?

### DISCUSSION THREAD

**Discussion by TheTechRobo** [PiSupply/PiJuice#958] [Q&A]
I wanted to try the Compute Module 3 for my project with a custom PCB for a slim touchscreen device. I'm fairly new to this kind of thing, though, and I've already purchased a PiJuice for the project, so is the PiJuice compatible with the Compute Module 3, and if so, what do I need to make it work?

Thanks in advance!

**Comment by tvoverbeek** [comment]
I do not beiieve it is compatible, due to the number of different power supplies needed for the CM3.
See the CM3 datasheet.Will also depend on the type of IO board you use to mount the CM3.
Power and GPIO connections are very different from the Raspberry Pi SBCs

**Reply by TheTechRobo** [reply]
Thanks, do you have a recommendation for other boards?

**Comment by tvoverbeek** [comment]
Any board which has a compatible 40-pin GPIO header (power pins, ground and I2C are used by the PiJuice)

**Reply by TheTechRobo** [reply]
Oh, I meant other boards similar to PiJuice - other ones.

I've been thinking of using a LiFePO4 battery, so it'd need to have support for that too.

(But I'll probably just stick with the thickness of the Pi3.)

**Comment by TheTechRobo** [comment]
Oh, by the way, this is slightly off-topic, but what does happen if you plug a multi-cell battery into PiJuice? I ask because I dont know much about batteries and it isn't clear to me whether [this](https://www.pishop.ca/product/lifepo4-battery-3-2v-12ah/) is single-cell or multi-cell. (Also, I don't think that has one of those temperature sensor wires; could disabling PiJuice's temperature sensor cause overheating?)
